fares-data-build-tool/create-data
# feat: Edit Carnet tickets number in bundle and carnet duration

## Description

For a carnet ticket, the user should be able to edit the number of tickets in the bundle, via productDetails page.

## Testing instructions

Make a carnet type product and then go to the appropriate services/other products page and select that carnet product
edit the number in bundle and expiry

Try different valid/invalid combinations

Run the myFare.cy.ts everything should pass

Run the unit tests for editCarnetProperties.test.tsx and .ts
